Debian/bin/lib/Cyrus/Docker/Command/build.pm

```diff
@@ -41,7 +41,7 @@ sub execute ($self, $opt, $args) {
 
   $self->configure($opt) unless $opt->recompile;
 
-  my @jobs = ("-j", $opt->jobs);
+  my @jobs = ("-j", $self->app->config->{default_jobs} // $opt->jobs);
```

Comment:
Sure, though might be nice to be sure default_jobs is an integer. The Getopt::Long... parser makes sure `$opt->jobs` is, but this doesn't

Reply:
So maybe do some validation in `sub configure`. (Bonus points if validation can be triggered separately so `dar start` can validate the file when it copies it in too)

Reply:
I think `configure` is the wrong place, since `test` will also use this.  I'll put it into config loading.

Reply:
Heh that's what I meant, something something overloaded names